app/src/sharedTest/java/org/oppia/app/player/audio/AudioFragmentTest.kt

```diff
@@ -160,7 +160,7 @@ class AudioFragmentTest {
         PROFILE_ID_DEFAULT_AUDIO_LANGUAGE_HINDI
       )
     ).use {
-      onView(withId(R.id.tvAudioLanguage)).check(matches(withText("HI")))
+      onView(withId(R.id.tvAudioLanguage)).check(matches(isDisplayed()))
```

Comment:
It seems like all of these 3 test cases are checking the same thing, is `tvAudioLanguage` visible or not. 

We can convert it to one test case mentioned below and remove the other three. 

```
@Test
  fun testAudioFragment_openFragment_showsAudioLanguageIcon() {
    launch<AudioFragmentTestActivity>(
      createHomeActivityIntent(
        PROFILE_ID_DEFAULT_AUDIO_LANGUAGE_ENGLISH
      )
    ).use {
      onView(withId(R.id.tvAudioLanguage)).check(matches(isDisplayed()))
    }
  }
```

Reply:
In case of test case not passing, please mention the below todo and annotation at the top of your test cases. 

```
  // TODO(#973): Fix AudioFragmentTest
  @Ignore
```